projects/js-packages/charts/src/components/shared/with-responsive.tsx

```diff
@@ -23,7 +23,7 @@ export function withResponsive< T extends BaseChartProps< unknown > >(
 ) {
 	const { maxWidth = 1200, aspectRatio = 0.5, debounceTime = 50 } = config || {};
```

Comment:
I'm wondering where can we change those params 🤔

Reply:
hmm... okay so they are passed via the config prop in the component. I guess a 0.5 aspectRatio is a bit weird for a default as it's not super relevant to anything other than the semi circle chart. What specifically are you interested in changing?

Reply:
`aspectRatio`